plugin/storage/es/spanstore/reader_test.go

```diff
@@ -118,6 +120,7 @@ func withArchiveSpanReader(readAlias bool, fn func(r *spanReaderTest)) {
 		reader: NewSpanReader(SpanReaderParams{
 			Client:              client,
 			Logger:              zap.NewNop(),
+			Tracer:              jtracer.NoOp().OTEL.Tracer("test"),
```

Comment:
how will we know if the traces are created if it's noop? You already went through this with metricsreader PR